src/youtube-tracker.coffee

```diff
@@ -14,9 +13,42 @@
 # Author:
 #   Karl Birch <mbwkarl@gmail.com>
 
+GOOGLE_API_KEY = process.env.GOOGLE_API_KEY
+
+querystring = require 'querystring'
+url = require 'url'
```

Comment:
It seems like we're using Hubot's own HTTP library, are these dependencies really needed?